src/utils/timing.test.ts

```diff
@@ -6,11 +6,30 @@ import { IntervalPoller, pauseWithJitter } from "./timing";
 /** mocha tests over function pauseWithJitter */
 describe("pauseWithJitter", () => {
   it("should pause", async () => {
-    const start = Date.now();
-    await pauseWithJitter(20, 50);
-    // Give a little bit more padding for CI.
-    const elapsed = Date.now() - start;
-    assert.strictEqual(elapsed >= 20 && elapsed <= 55, true);
+    const clock = sinon.useFakeTimers();
```

Comment:
Debated using a sandbox for this suite, but shouldn't be needed unless other tests similarly start acting up and need timers (or a sandbox for other reasons)